spec/google/gax/api_callable_spec.rb

```diff
@@ -93,15 +117,27 @@ module Gax
     end
 
     it 'iterates over elements' do
-      my_callable = Google::Gax.create_api_call(func, settings)
+      func2 = proc do |request, **kwargs|
```

Comment:
Can you just change `func` to return a mocked `op` instead of wrapping it in `func2` (here and below)?